.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Feature request or idea? Consider opening an [API review](.github/API_REVIEW.md)!
```

Comment:
Should this be a comment?

Reply:
I had this uncommented so that it would render to a clickable link, and also be visible if you navigate to the rendered ISSUE_TEMPLATE.md file directly.